samples/excel/30-range/test-for-used-range.yaml

```diff
@@ -0,0 +1,125 @@
+id: excel-test-for-used-range
+name: Test for used range
+description: Create a chart from a table only if there's data in the table.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#try-create-chart-from-empty-table").click(() => tryCatch(tryCreateChartFromEmptyTable));
+
+        async function tryCreateChartFromEmptyTable() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");    
+                const salesTable = sheet.tables.getItem("SalesTable");
+                const dataRange = salesTable.getDataBodyRange();
+                const usedDataRange = dataRange.getUsedRangeOrNullObject(true /* only cells with values count as used */);
+
+                //Must sync before reading value returned from *OrNullObject method/property.
+                await context.sync();
+
+                if (!usedDataRange.isNullObject) {
+                    const chart = sheet.charts.add("ColumnClustered", dataRange, "auto");
+                    chart.setPosition("A15", "F30");
+                    chart.title.text = "Quarterly sales chart";
+                    chart.legend.position = "right"
+                    chart.legend.format.fill.setSolidColor("white");
+                    chart.dataLabels.format.font.size = 15;
+                    chart.dataLabels.format.font.color = "black";
+                } else {            
+                    OfficeHelpers.UI.notify("To create a meaningful chart, add names to the Product column and numbers to some of the other cells. Then press 'Try to create chart' again.");
```

Comment:
Notify has a method that takes in a title and a string.  I recommend you use that one.  It just looks nicer...